### ISSUE TITLE
paper-tabs accessibility report

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-tabs#4]
Here are the accessibility developer tools audit results: 

[Warning] Elements with onclick handlers must be focusable (8)[Warning] Text elements should have a reasonable contrast ratio (33)Not applicable tests (9)Passing tests (4)

Major issues: 

Contrast: Contrast needs to be checked, especially for the items that are currently not chosen. I know the point is for them to be less visible to indicate that they are not chosen, but it's very hard to read the text. Also, the blue link text over the blue/green background is nearly impossible to read. All contrast must be reviewed here. 

Keyboard navigation and focus state:  I am not able to navigate to all the Items using just the keyboard and enter. Also, when I am using a screen reader (VoiceOver and ChromeVox) to navigate through, I am able to press the force click key command to actually click on the given item, but this is not triggering the same visual response as when I use the mouse to physically click. Using the mouse, I see a bigger visual animation, so the experience is inconsistent with keyboard vs. mouse. Happy to show you this in person. 

Note in regards to roles: Currently, a user using a screen reader would not really know that they could click on the various items to change focus, as there is no spoken feedback indicating that it is in fact clickable.  When they are implemented as links, this is different because the user can hear the earcon and spoken text that this is a link. We should brainstorm how we could identify this option to users for the other tab forms though (aside from the links).

**Comment by alice** [comment]
Ideally this should follow the ARIA design pattern for tablist/tab: 
http://www.w3.org/TR/wai-aria-practices/#tabpanel

**Comment by cdata** [comment]
ARIA improvements have landed, and screen readers are doing a much improved job of describing the tabs. I still have to address the contrast.

**Comment by lpalmaro** [comment]
Just tested again. The arrow button to scroll the tabs to the right doesn't have a label. It just says button. Also, the blue text on the blue/green background doesn't meet contrast guidelines. I think this is blue because they are meant to be links... Anything we can do here to make the contrast better?

**Comment by bicknellr** [comment]
Now that #99 is in and contrast seems to have been the only open problem in this issue, I'm going to close.
